libraries/AP_HAL_ChibiOS/hwdef/OrqaH743Pro/hwdef.dat

```diff
@@ -0,0 +1,186 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for H7PRO hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_ORQAH743PRO
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+MCU_CLOCKRATE_MHZ 480
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+FLASH_SIZE_KB 2048
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 384
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+# SPI devices
+
+# SPI1 - MPU6000
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2 - dataflash
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3
+PB3 SPI3_SCK SPI3
+PB4 SPI3_MISO SPI3
+PD6 SPI3_MOSI SPI3
+
+# SPI4 - ICM42605
+# PE12 SPI4_SCK SPI4
+# PE13 SPI4_MISO SPI4
+# PE14 SPI4_MOSI SPI4
+
+# Chip select pins
+PB12 FLASH1_CS CS
+PA15 OSD1_CS CS
+PA4 GYRO1_CS CS
+PE11 GYRO2_CS CS
+
+# Beeper
+PE9 TIM1_CH1 TIM1 GPIO(80) ALARM
+PE9 BUZZER OUTPUT GPIO(81) LOW ALT(1)
+
+# SERIAL ports
+SERIAL_ORDER OTG1 EMPTY EMPTY USART3 EMPTY EMPTY USART6 UART7 UART8 OTG2
+
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART3 - RX
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+# Default is GHST protocol
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_RCIN
+define DEFAULT_SERIAL3_OPTIONS 4
+
+# USART6 - Sik
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_MAVLink2
+
+# UART7 - GPS
+PE7 UART7_RX UART7 NODMA
+PE8 UART7_TX UART7 NODMA
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_GPS
+
+# UART8 - ESC Telemetry
+PE0 UART8_RX UART8 NODMA
+define DEFAULT_SERIAL8_PROTOCOL SerialProtocol_ESCTelemetry
+
+# CAN bus
+PB8 CAN1_RX CAN1
+PB9 CAN1_TX CAN1
+
+# I2C ports
+I2C_ORDER I2C1 I2C2
+# I2C1
+PB6 I2C1_SCL I2C1
+PB7 I2C1_SDA I2C1
+
+# I2C2
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+# Motors
+PD12 TIM4_CH1 TIM4 PWM(1) GPIO(50) BIDIR
+PD13 TIM4_CH2 TIM4 PWM(2) GPIO(51)
+PA1  TIM2_CH2 TIM2 PWM(3) GPIO(52) BIDIR
+PA0  TIM2_CH1 TIM2 PWM(4) GPIO(53)
+PA2  TIM5_CH3 TIM5 PWM(5) GPIO(54) BIDIR
+PA3  TIM5_CH4 TIM5 PWM(6) GPIO(55)
+PB1  TIM3_CH4 TIM3 PWM(7) GPIO(56) BIDIR
+PB0  TIM3_CH3 TIM3 PWM(8) GPIO(57)
+# Servos
+PE6  TIM15_CH2 TIM15 PWM(9)  GPIO(58) NODMA
+PE5  TIM15_CH1 TIM15 PWM(10) GPIO(59) NODMA
+
+# CAM switch
+PD0 CAM_SW OUTPUT GPIO(74) LOW
+define RELAY2_PIN_DEFAULT 74
+
+# GPIO 5
+PD14 PINIO1 OUTPUT GPIO(75) LOW
+
+# ADC ports
+
+# ADC1
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_VOLT_SCALE 11.0
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1)
+define HAL_BATT_CURR_PIN 11
+define HAL_BATT_CURR_SCALE 92.6
+define HAL_BATT_MONITOR_DEFAULT 4
+
+# LEDs
+PA8 LED0 OUTPUT LOW GPIO(90)
+define HAL_GPIO_B_LED_PIN 90
+PA10 LED1 OUTPUT LOW GPIO(91)
+define HAL_GPIO_A_LED_PIN 91
+define HAL_LED_ON 1
+define HAL_LED_OFF 0
```

Comment:
remove, not used anymore

Reply:
need to add
define AP_NOTIFY_GPIO_LED_2_ENABLED 1

Reply:
Fixed